networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/AttestationTopicSubscriber.java

```diff
@@ -97,6 +107,15 @@ public synchronized void subscribeToPersistentSubnets(
 
     for (SubnetSubscription subnetSubscription : newSubscriptions) {
       int subnetId = subnetSubscription.subnetId();
+      if (lastSlot.get() != null
+          && subnetSubscription.unsubscriptionSlot().isLessThan(lastSlot.get())) {
```

Comment:
same here, read lastSlot into a `UInt64` in preference to this so that we can call get once